pkg/kudoctl/packages/install/install.go

```diff
@@ -0,0 +1,144 @@
+// Package install provides function to install package resources
+// on a Kubernetes cluster.
+package install
+
+import (
+	"strings"
+	"time"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+type Options struct {
+	skipInstance    bool
+	wait            *time.Duration
+	createNamespace bool
+}
+
+type Option func(*Options)
+
+// SkipInstance installs only Operator and OperatorVersion
+// of an operator package.
+func SkipInstance() Option {
```

Comment:
It's an interesting pattern but I wonder why we need it. Why not simply pass `install.Options` struct to the `install.Package` method and let the caller set the options that are needed?

Reply:
To provide a backwards-compatible way of declaring options. E.g. the underlying option types could be changes without voiding these functions -- only changing their implementations. But that's probably a bit too much, as it also complicates how that function is called in `installOperator`. I'll change this to use an `install.Options`.